database/sql/V055__create_settlement_call_data.sql

```diff
@@ -0,0 +1,10 @@
+-- Add non-optional `simulation_block` field to settlement_scores table. 
+ALTER TABLE settlement_scores ADD COLUMN simulation_block bigint NOT NULL DEFAULT 0;
+ALTER TABLE settlement_scores ALTER COLUMN simulation_block DROP DEFAULT;
+
+-- Winning settlement call data for each auction.
+CREATE TABLE settlement_call_data (
```

Comment:
Please don't forget to update the [`database/README.md`](https://github.com/cowprotocol/services/blob/main/database/README.md) when altering the DB schema.

Reply:
Ah yees. Added.